docs/com1DFAAlgorithm.rst

```diff
@@ -166,6 +166,10 @@ Other particles properties are also initialized here:
     
     - ``dmDet`` - detrained mass of particle [kg]
 
+    - ``dmEnt`` - entrained mass of particle [kg]
+
+    - ``stoppedParticles`` - particles that are stopped (mass or velocity is zero) and deleted from the particles
```

Comment:
maybe add more info, that this is a dictionary containing information like x, y... for each time step right?